feathr_project/feathr/_feature_registry.py

```diff
@@ -791,12 +813,15 @@ def get_features_from_registry(self, project_name: str, workspace_path: str):
 
         with open(feature_conf_file, "w") as features:
             features.write(feature_conf_content)
-        logger.info("Writing feature configuration from feathr registry to {}", feature_conf_file)
-        
+        logger.info(
+            "Writing feature configuration from feathr registry to {}", feature_conf_file)
+
         with open(feature_join_file, "w") as offline_config:
             offline_config.write(feature_join_conf_content)
-        logger.info("Writing offline configuration from feathr registry to {}", feature_join_file)
+        logger.info(
+            "Writing offline configuration from feathr registry to {}", feature_join_file)
```

Comment:
Is the indentation expected? They seem to be inconsistent with other code. Did you do this manually, or with some plugin/automatically.

Reply:
Ah, must just be the formatter running in my IDE - I can revert that

Reply:
Updated my PR to match existing formatting in the repo